phpunit.xml.dist

```diff
@@ -1,32 +1,69 @@
+	 - "vendor/bin/phpunit <dash><dash>coverage-html assets/": Generate coverage report (replace <dash> with "-", requires xdebug)
 	More information:
 	- http://www.phpunit.de/manual/current/en/textui.html
 	- http://doc.silverstripe.org/framework/en/topics/testing/#configuration
-
 	It is safe to remove this file for normal website operation.
 -->
-<phpunit bootstrap="framework/tests/bootstrap.php" colors="true">
+<phpunit bootstrap="vendor/silverstripe/framework/tests/bootstrap.php" colors="true">
 
 	<testsuite name="Default">
 		<directory>mysite/tests</directory>
-		<directory>cms/tests</directory>
-		<directory>framework/tests</directory>
+		<directory>vendor/silverstripe/subsites/tests</directory>
+		<directory>vendor/silverstripe/registry/tests</directory>
+		<directory>vendor/tractorcow/silverstripe-fluent/tests</directory>
 	</testsuite>
-	
-	<listeners>
-		<listener class="SS_TestListener" file="framework/dev/TestListener.php" />
-	</listeners>
+
+    <!-- cwp core components -->
+	<testsuite name="cwpcore">
+		<directory>vendor/cwp/cwp-core/tests</directory>
+		<directory>vendor/silverstripe/auditor/tests</directory>
+		<directory>vendor/silverstripe/environmentcheck/tests</directory>
+		<directory>vendor/silverstripe/hybridsessions/tests</directory>
+		<directory>vendor/silverstripe/mimevalidator/tests</directory>
+	</testsuite>
+
+	<testsuite name="cwpcms">
+		<directory>vendor/cwp/cwp/tests</directory>
+		<directory>vendor/silverstripe/taxonomy/tests</directory>
+		<directory>vendor/silverstripe/html5/tests</directory>
+		<directory>vendor/cwp/cwp-pdfexport/tests</directory>
+		<directory>vendor/symbiote/silverstripe-gridfieldextensions/tests</directory>
+	</testsuite>
+
+	<testsuite name="cwpsearch">
+		<directory>vendor/cwp/cwp-search/tests</directory>
+		<directory>vendor/silverstripe/fulltextsearch/tests</directory>
+		<directory>vendor/symbiote/silverstripe-queuedjobs/tests</directory>
+		<directory>vendor/silverstripe/textextraction/tests</directory>
+	</testsuite>
+
+    <!-- silverstripe core components -->
+    <testsuite name="core">
+		<directory>vendor/silverstripe/framework/tests/php/</directory>
+		<directory>vendor/silverstripe/config/tests</directory>
+		<directory>vendor/silverstripe/assets/tests/php/</directory>
+        <directory>vendor/silverstripe/versioned/tests/php/</directory>
+    </testsuite>
+
+    <!-- silverstripe admin components -->
+    <testsuite name="admin">
+        <directory>vendor/silverstripe/cms/tests/</directory>
+        <directory>vendor/silverstripe/admin/tests/php/</directory>
+        <directory>vendor/silverstripe/campaign-admin/tests/php/</directory>
+        <directory>vendor/silverstripe/asset-admin/tests/php/</directory>
+        <directory>vendor/silverstripe/graphql/tests/</directory>
+        <directory>vendor/silverstripe/siteconfig/tests/php/</directory>
+        <directory>vendor/silverstripe/reports/tests/</directory>
+    </testsuite>
```

Comment:
The indentation is a bit inconsistent here - mind switching it all to spaces?

Reply:
Done :)